imageio/plugins/tifffile.py

```diff
@@ -74,6 +75,10 @@ class TiffFormat(Format):
     was written has no effect. Also, global metadata is ignored if metadata is
     provided via the `meta` argument of :py:meth:`Format.Writer.append_data`.
 
+    If you have installed tifffile as a Python package, imageio will attempt
+    to use that instead of the bundled version from this plugin. Doing so can
```

Comment:
```suggestion
    to use that as backend instead of the bundled backend. Doing so can
```

same as above refering to the `tifffile` module as backend should avoid ambiguity.